omnibus/package-scripts/installer-rpm/prerm

```diff
@@ -4,17 +4,14 @@
 
 readonly INSTALL_DIR=/opt/datadog-installer
 
-SYSTEMCTL_SKIP_SYSV=true systemctl stop datadog-installer || true
-SYSTEMCTL_SKIP_SYSV=true systemctl disable datadog-installer || true
```

Comment:
Don't you need to at least stop the units before removing them as part of `installer purge`?

Reply:
I was pretty certain that purge itself was stopping the units but apparently it doesn't yet.
I added the stop in the installer purge